ble_controller/include/ble_controller_hci_vs.h

```diff
@@ -0,0 +1,216 @@
+};
+
+/** @} end of HCI_VS_TYPES */
+
+/**
+ * @defgroup HCI_VS_EVENTS Events
+ * @{
+ */
+
+/** @brief QoS Connection Event Report.
+ *
+ * A QoS Connection Event report gives information about the connection event.
+ */
+typedef __PACKED_STRUCT
+{
+    /** @brief Connnection handle corresponding to the connection event report. */
+    uint16_t conn_handle;
+    /** @brief Connection event counter corresponding to the connection event report. */
+    uint16_t event_counter;
+    /** @brief Data Channel Index used during the connection event (0-36). */
+    uint8_t channel_index;
+    /** @brief Number of packets received with good CRC during the connection event. */
+    uint16_t crc_ok_count;
+    /** @brief Number of packets received with bad CRC during the connection event. */
+    uint8_t crc_error_count;
+    /** @brief Indicates that the connection event was closed because a packet was not received. */
+    uint8_t rx_timeout;
+} hci_vs_evt_qos_conn_event_report_t;
+
+/** @} end of HCI_VS_EVENTS */
+
+/**
+ * @defgroup HCI_VS_COMMANDS Commands
+ * @{
+ */
+
+/** @brief Set Low Latency Packet Mode command parameter(s). */
+typedef __PACKED_STRUCT
+{
+    /** @brief Set to 1 to enable LLPM. */
+    uint8_t enable;
+} hci_vs_cmd_llpm_mode_set_t;
+
+/** @brief Connection Update command parameter(s). */
+typedef __PACKED_STRUCT
+{
+    /** @brief Connection Handle. */
+    uint16_t connection_handle;
+    /** @brief Connection Interval in microseconds. Valid range is 7,500 us to 4,000,000 us in 1,250
+     *         us steps.  If LLPM mode is enabled, parameters in the range 1,000 us to 7,000 us in
+     *         1,000 us steps are also accepted.
+     */
+    uint32_t conn_interval_us;
+    /** @brief Slave latency for the connection in number of connection events. */
+    uint16_t conn_latency;
+    /** @brief Supervision timeout for the LE Link in 10 ms units. Range 100 ms to 32 ms. */
+    uint16_t supervision_timeout;
+} hci_vs_cmd_conn_update_t;
+
+/** @brief Enable or Disable Extended Connection Events command parameter(s). */
+typedef __PACKED_STRUCT
+{
+    /** @brief Set to 0 for disabling, 1 for enabling, all other values are RFU. */
+    uint8_t enable;
+} hci_vs_cmd_conn_event_extend_t;
+
+/** @brief QoS Connection Event Reports enable command parameter(s). */
+typedef __PACKED_STRUCT
+{
+    /** @brief Set to 0 for disabling, 1 for enabling, all other values are RFU. */
+    uint8_t enable;
+} hci_vs_cmd_qos_conn_event_report_enable_t;
+
+/** @} end of HCI_VS_COMMANDS */
+
+/**
+ * @defgroup HCI_VS_API API
+ * @{
+ */
+
+/** @brief Set Low Latency Packet Mode.
+ *
+ * This command enables or disables Low Latency Packet Mode support.
+ * When Low Latency Packet Mode is enabled, it is possible to switch to connection intervals in the
+ * range 1-7 ms. Switch to short connection intervals by calling HCI_LE_Connection_Update
+ * with Conn_Interval set to (0x0D00 + conn_interval_ms).
```

Comment:
Which one should be used: HCI_LE_Connection_Update or hci_vs_cmd_conn_update

Reply:
Will switch to `@ref hci_vs_cmd_conn_update()` and "hide" the implementation detail
with `Conn_Interval set to (0x0D00 + conn_interval_ms)`